controllers/apply/form-router-next/index.js

```diff
@@ -69,10 +73,44 @@ function initFormRouter({
         next();
     }
 
-    router.use(csrfProtection, injectCopy('applyNext'), setCommonLocals);
+    router.use(
+        // Populate req.body for multipart forms before CSRF token is needed
+        (req, res, next) => {
```

Comment:
It might be worth moving this to the step handler, just to really limit where this is needed, but doesn't need to happen in this PR.